src/frame.c

```diff
@@ -355,7 +352,8 @@ static void aas_push(frame_t *st, uint8_t* psd, unsigned int length, logical_cha
     }
     else if (fcs16(psd, length) != VALIDFCS16)
     {
-        log_info("psd crc mismatch");
+        // occasional CRC errors are normal, because transmitters abandon their HDLC
+        // frame mid-stream when switching to new PSD data
```

Comment:
As noted, these CRC mismatches originate from the transmitter, so it doesn't make sense to warn about them.